offgridplanner/projects/models.py

```diff
@@ -110,17 +111,25 @@ def filter_consumers(self, consumer_type):
 
     @property
     def counts(self):
-        counts = self.node_df.groupby(['consumer_type', 'consumer_detail']).size()
+        counts = self.df.groupby(['consumer_type', 'consumer_detail']).size()
         return counts
 
     @property
     def have_custom_machinery(self):
-        machinery = self.node_df.groupby(['consumer_type', 'consumer_detail']).agg({'custom_specification': ';'.join}).custom_specification.loc["enterprise"]
+        machinery = self.df.groupby(['consumer_type', 'consumer_detail']).agg({'custom_specification': ';'.join}).custom_specification.loc["enterprise"]
         if not machinery.eq("").all():
             return True
         else:
             return False
 
+class Links(models.Model):
```

Comment:
NB: for future we should have many links linked to one project and not one object with a single DB entry as JSONField